osu.Game.Tests/Visual/UserInterface/TestSceneModSettings.cs

```diff
@@ -53,6 +55,17 @@ public void TestButtonShowsOnModAlreadyAdded()
             AddAssert("button enabled", () => modSelect.CustomiseButton.Enabled.Value);
         }
 
+        [Test]
+        public void TestCustomisationOpensOnModSelect()
```

Comment:
This test case needs more work:

* Need to also test that deselecting all mods after selecting with customisation hides the settings
* Running this test twice will cause it to fail the second time since the customisation mod remains selected between test runs

Could you describe what sort of issues you're having with the running tests so we can help resolve them? It's not really a good idea to write tests blindly without being able to run them, kind of defeats the purpose of the entire endeavour.

Reply:
So interestingly enough, tests work fine on Windows and don't work on Linux for me, unless I'm doing it wrong.
To run the tests I used `dotnet test "/home/protheory8/osu/osu.Game.Tests/osu.Game.Tests.csproj"` and the same command on Windows with a different path.

The output on Linux:
`Test run for /home/protheory8/osu/osu.Game.Tests/bin/Debug/netcoreapp3.1/osu.Game.Tests.dll`
`(.NETCoreApp,Version=v3.1)`
`Microsoft (R) Test Execution Command Line Tool Version 16.3.0`
`Copyright (c) Microsoft Corporation.  All rights reserved.`

`Starting test execution, please wait...`

`A total of 1 test files matched the specified pattern.`
`  ! TestParity("Soleily - Renatus (Gamu) [Insane].osu")`
`  ! TestParity("Within Temptation - The Unforgiving (Armin) [Marathon].osu")`

`The active test run was aborted. Reason: Test host process crashed`

`Test Run Aborted.`
`Total tests: Unknown`
`     Passed: 77`
`    Skipped: 2`
` Total time: 20.5240 Seconds`

Reply:
If there's no exception provided, you may want to run with the `-v diag` option.

Reply:
I recall something in `libbassfx` segfaulting on me when trying to run tests headless on Linux (outside of test browser), but never tried to diagnose in detail. You can check `dmesg` output to confirm.